train_configs/sft/default.yaml

```diff
@@ -0,0 +1,20 @@
+model_name_or_path: allenai/OLMo-1.7-1B-hf
+model_revision: main
+use_flash_attn: true
+tokenizer_name: allenai/OLMo-1.7-1B-hf
+use_slow_tokenizer: true
+train_file: data/processed/tulu_v2/tulu_v2_data.jsonl
```

Comment:
could we just swap this to point to the hf uploaded version? makes it even more portable :)

Reply:
Yes fixing this, was a typo / example for debugging not actual training.